app/models/banner.rb

```diff
@@ -0,0 +1,2 @@
+class Banner < ActiveRecord::Base
```

Comment:
I recommend adding the message presence validation here too.

Reply:
Ah. Done!